src/App.tsx

```diff
@@ -1,7 +1,12 @@
 import "./App.css";
+import Styledropdown from "./components/style_dropdown";
 
 function App() {
-  return <p>Wilshire Hospice App</p>;
+  return (
```

Comment:
Once you are done with this issue, please revert App.tsx back to its original state.